### ISSUE TITLE
Mongo Error when the RESERVED Key was used as a model attribute

### ISSUE THREAD

**Issue by rasekar2000** [balderdashy/sails-mongo#287]
When the reserved keyword is used as a model attribute then the error was not clear.
<code> module.exports = {
  attributes: {
      name:{
        type:"string", 
        required:true,
        minLength: 3
      },
      <b>type</b>:{
        type:"string",
        unique: false
      }
} </code>

verbose: Grunt :: Running "watch" task
verbose: Grunt :: Waiting...
error: Sending 500 ("Server Error") response: 
 Error (E_UNKNOWN) :: Encountered an unexpected error
MongoError: topology was destroyed
    at Server.insert (/Volumes/DataPartition/sails/test7REST/node_modules/sails-mongo/node_modules/mongodb/node_modules/mongodb-core/lib/topologies/server.js:894:49)
    at Server.insert (/Volumes/DataPartition/sails/test7REST/node_modules/sails-mongo/node_modules/mongodb/lib/server.js:311:17)
    at insertDocuments (/Volumes/DataPartition/sails/test7REST/node_modules/sails-mongo/node_modules/mongodb/lib/collection.js:493:19)
    at Collection.insert (/Volumes/DataPartition/sails/test7REST/node_modules/sails-mongo/node_modules/mongodb/lib/collection.js:535:10)
    at Collection.insert (/Volumes/DataPartition/sails/test7REST/node_modules/sails-mongo/lib/collection.js:182:48)
    at Object.module.exports.adapter.create (/Volumes/DataPartition/sails/test7REST/node_modules/sails-mongo/lib/adapter.js:274:18)
    at module.exports.create (/usr/local/lib/node_modules/sails/node_modules/waterline/lib/waterline/adapter/dql.js:85:13)
    at bound.createValues (/usr/local/lib/node_modules/sails/node_modules/waterline/lib/waterline/query/dql/create.js:213:16)
    at /usr/local/lib/node_modules/sails/node_modules/waterline/lib/waterline/query/dql/create.js:74:20
    at /usr/local/lib/node_modules/sails/node_modules/async/lib/async.js:254:17
    at /usr/local/lib/node_modules/sails/node_modules/async/lib/async.js:157:25
    at /usr/local/lib/node_modules/sails/node_modules/async/lib/async.js:251:21
    at /usr/local/lib/node_modules/sails/node_modules/async/lib/async.js:615:34
    at /usr/local/lib/node_modules/sails/node_modules/async/lib/async.js:157:25
    at bound.<anonymous> (/usr/local/lib/node_modules/sails/node_modules/waterline/lib/waterline/utils/schema.js:151:44)
    at fn (/usr/local/lib/node_modules/sails/node_modules/waterline/lib/waterline/utils/callbacksRunner.js:41:10)

Details:  MongoError: topology was destroyed

------------------------------------------------------------------------
There are no issues during the Sails Lift but fails during the Sails Data insert using Blueprint - Create.

I think a better error handling can be done without crashing the server. 

Thanks in advance.

**Comment by sailsbot** [comment]
Thanks for posting, @rasekar2000.  I'm a repo bot-- nice to meet you!

It has been 30 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
